### PR TITLE
Try spot bugs issue on PR 50

### PR DESCRIPTION
**Pull request by jeffret-b**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/audit/listeners/SaveableChangeListener.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    @Override
+    public void onChange(Saveable o, XmlFile file) {
+
+        if(o instanceof Fingerprint){
+            UseCredentials useCredentials = LogEventFactory.getEvent(UseCredentials.class);
+            Fingerprint fp = (Fingerprint) o;
+            useCredentials.setFileName(fp.getFileName());
+            useCredentials.setName(fp.getDisplayName());
+            useCredentials.setTimestamp(formatDateISO(fp.getTimestamp().getTime()));
+            if (fp != null) {
```
Comment (jvz) [inline src/main/java/io/jenkins/plugins/audit/listeners/SaveableChangeListener.java]: Impossible due to line 28, and spotbugs should know that.

Comment (jeffret-b) [reply src/main/java/io/jenkins/plugins/audit/listeners/SaveableChangeListener.java]: It should. But before that change spotbugs failed locally and now passes. Something is wrong with spotbugs, but I'm not confident I'm understanding what's going on yet.

Comment (jeffret-b) [reply src/main/java/io/jenkins/plugins/audit/listeners/SaveableChangeListener.java]: Now it realizes the null check is redundant.

**Comment by jeffret-b** [comment]
No longer useful

**Pull request closed without merging**